source/includes/_applications.md

```diff
@@ -369,3 +369,33 @@ In order to delete a closed or hired Job Application you must use the following
 `DELETE /v1/companies/{companyId}/applications/{jobApplicationId}`
 
 If the request succeeds, the api will return a 200 response with no body.
+
+## Edit a JobApplication
+
+This endpoint is used to edit a job application's information. It can also be used to edit and delete associated files and profile pictures from storage.
```

Comment:
This endpoint is used to edit a job application's information. It can also be used to edit or delete associated files and profile pictures from storage.